vertx-template-engines/vertx-web-templ-handlebars/src/main/java/io/vertx/ext/web/templ/impl/JsonObjectValueResolver.java

```diff
@@ -0,0 +1,42 @@
+package io.vertx.ext.web.templ.impl;
+
+import java.util.Collections;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import com.github.jknack.handlebars.ValueResolver;
+
+import io.vertx.core.json.JsonObject;
+
+public enum JsonObjectValueResolver implements ValueResolver {
+
```

Comment:
Most of the Handlebars resolvers also applythat pattern thus I did not want to deviate from the other implementations. I can change it if you want.